language/analyze.js

```diff
@@ -12,163 +12,294 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+'use strict';
+
+// [START all]
+// [START setup]
+// By default, the client will authenticate using the service account file
+// specified by the GOOGLE_APPLICATION_CREDENTIALS environment variable and use
+// the project specified by the GCLOUD_PROJECT environment variable. See
+// https://googlecloudplatform.github.io/gcloud-node/#/docs/google-cloud/latest/guides/authentication
+var Language = require('@google-cloud/language');
+var Storage = require('@google-cloud/storage');
+
+// Instantiate the language client
+var language = Language();
+// Instantiate the storage client
+var storage = Storage();
+// [END setup]
+
+// [START analyze_sentiment]
 /**
- * @fileoverview Command-line program to demonstrate how to call different
- * methods in Cloud Natural Language API.
- *
- * To run this example, install npm:
- *   npm install
+ * Detect the sentiment of a block of text.
  *
- * You must also set up to authenticate with the Cloud APIs using your
- * project's service account credentials. See the README for details.
- *
- * To run:
- *   node analyze.js <sentiment|entities|syntax> <text>
- *
- * Here is an example:
- *   node analyze.js entities "President Obama is speaking at the White House."
+ * @param {string} text The text to analyze.
+ * @param {object} [options] Configuration options.
+ * @param {string} [options.type] "text" or "html".
+ * @param {string} [options.language] The language of the text, e.g. "en".
+ * @param {function} callback The callback function.
  */
-'use strict';
+function analyzeSentiment (text, options, callback) {
+  var document = language.document({
+    content: text,
+    type: options.type,
+    language: options.language
+  });
 
-var google = require('googleapis');
+  var config = {
+    // Get more detailed results
+    verbose: true
+  };
 
-var languageScopes = ['https://www.googleapis.com/auth/cloud-platform'];
+  // See https://googlecloudplatform.github.io/google-cloud-node/#/docs/language/latest/language/document
+  document.detectSentiment(config, function (err, sentiment) {
+    if (err) {
+      return callback(err);
+    }
 
+    console.log('Found %s sentiment', sentiment.polarity >= 0 ? 'positive' : 'negative');
```

Comment:
Both polarity and magnitude will get printed by the CLI. I don't think the number itself provides that much information beyond being positive or negative, at least not to me.